File: All-Services/ClientApp/src/Model/Company/Company.ts
import { User } from "src/Model/User";
import { Telephones } from "src/Model/Company/Telephones";
import { CompanyAddress } from "src/Model/Company/CompanyAddress";
import { CompanyServiceType } from "src/Model/Company/CompanyServiceType";

export class Company {
  constructor(
    public id?: number,
    public userId?: number,
    public name?: string,
    public email?: string,
    public logoUrl?: string,
    public webSiteUrl?: string,
    public description?: string,
    public smallDescription?: string,
    public popularity?: number,
    public likes?: number,
    public dislikes?: number,
    public user?: User,
    public telephones?: Telephones[],
    public companyAddress?: CompanyAddress[],
    public companyServiceType?: CompanyServiceType[],
    public logoUrlFromServer?: string
  ) { }
}

File: AllServicesData/Model/Company.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AllServicesData.Model
{
    public partial class Company : EntityBase
    {
        public int UserId { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string LogoUrl { get; set; }
        public string WebSiteUrl { get; set; }
        public string Description { get; set; }
        public string SmallDescription { get; set; }
        public int Popularity { get; set; }
        public int Likes { get; set; }
        public int Dislikes { get; set; }
        public virtual User User { get; set; }
        public virtual ICollection<Telephones> Telephones { get; set; }
        public virtual ICollection<CompanyAddress> CompanyAddresses { get; set; }
        public virtual ICollection<CompanyImage> CompanyImages { get; set; }
        public virtual ICollection<CompanyServiceType> CompanyServiceType { get; set; }
    }
}
